boards/riscv/hifive1/hifive1_defconfig

```diff
@@ -10,7 +10,8 @@ CONFIG_UART_SIFIVE_PORT_0=y
 CONFIG_UART_CONSOLE=y
 CONFIG_PLIC=y
 CONFIG_PINMUX=y
```

Comment:
pinmux driver seems to be removed, so it should be this option

Reply:
Done.